backup_and_restore/application_backup_and_restore/troubleshooting.adoc

```diff
@@ -28,6 +28,29 @@ include::modules/velero-obtaining-by-accessing-binary.adoc[leveloffset=+1]
 include::modules/oadp-debugging-oc-cli.adoc[leveloffset=+1]
 include::modules/migration-debugging-velero-resources.adoc[leveloffset=+1]
 
+[id="oadp-pod-crash-resource-request"]
+== Pods crash or restart due to lack of memory or CPU
+
+If a Velero or Restic pod crashes due to a lack of memory or CPU, you can set specific resource requests for either of those resources.
```

Comment:
It seems like crash works in the context, so this is up to you! [IBMSG suggest](https://www.ibm.com/docs/en/ibm-style?topic=word-usage#crash) using a more specific term for `crash` like `fail` 
As well as the[ RHSS](https://redhat-documentation.github.io/supplementary-style-guide/#_c)

Reply:
Done.